build-system/sources.js

```diff
@@ -126,8 +127,16 @@ const CLOSURE_SRC_GLOBS = [
   '!node_modules/core-js/modules/library/**.js',
 ].concat(COMMON_GLOBS);
 
+let destination =
+  typeof argv.destination === 'string' ? argv.destination : './dist/';
```

Comment:
So if I understand it correctly the `argv.destination` will be declared in the release script where you will be calling `gulp dist`. I think that's good.
What are we setting the `argv.destination` to?
I think this is good. But are we planning on fix the destination based on experiment channel? Or do we want the experiment to define their own destination folder? Thanks